lib/loadtest.js

```diff
@@ -139,14 +139,14 @@ class Operation {
 	 */
 	startClients() {
 		const url = this.options.url;
+		const strBody = JSON.stringify(this.options.body);
```

Comment:
Please move this line inside the `if(this.options.body)` below, so as to avoid penalizing all users.